docs/excel/custom-functions-web-reqs.md

```diff
@@ -127,6 +126,9 @@ function increment(incrementBy, invocation) {
 }
 ```
 
+> [!NOTE]
```

Comment:
Later in this article is a note that says all cancelable functions must return 1 value. This sample returns void. So I'm not sure if the note needs an update.

Reply:
It seems like we each interpret this note differently. Perhaps we can sync offline and then adjust the note accordingly.